chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -468,7 +468,7 @@
 
         <!-- Pending Change Request Job - by ONC-ACB -->
         <job>
-            <name>Pending Change Request Report</name>
+            <name>Pending "Change Request" Report</name>
```

Comment:
Just curious - where was the request to rename this job? I didn't see a note about it in the ticket description nor did I see a test case to check it.

Reply:
This fixes [#OCD-3525], and the third commit tags that ticket. The ticket is in the backlog, and is the issue that we have with the code going to PROD today. I could remove this, but I was investigating it while in this code and saw this as an easy fix to an issue. When OCD-3525 comes up we'll be able to mark it as "cannot reproduce"

Reply:
I consider it part of the "leave it better than you found it" principle